server.js

```diff
@@ -1,21 +1,27 @@
-const express = require("express");
+const express = require('express');
+const ChatRoom = require('./templates/ChatRoom').ChatRoom;
```

Comment:
The "templates" are similar to "React" in the sense that you provide props, and perhaps even other functions (templates) to the template.